crowdstrike/table_crowdstrike_detect.go

```diff
@@ -0,0 +1,222 @@
+package crowdstrike
+
+import (
+	"context"
+
+	"github.com/crowdstrike/gofalcon/falcon"
+	"github.com/crowdstrike/gofalcon/falcon/client"
+	"github.com/crowdstrike/gofalcon/falcon/client/detects"
+	"github.com/crowdstrike/gofalcon/falcon/models"
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+	"golang.org/x/time/rate"
+)
+
+//// TABLE DEFINITION
+
+func tableCrowdStrikeDetects(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "crowdstrike_detect",
```

Comment:
```suggestion
		Name:        "crowdstrike_detection",
```

Would this table name be better as detection instead of detect? The API endpoint is `/detects/`, but the resource name seems to be referred to as detection(s) more commonly.